mkdocs.yml

```diff
@@ -18,11 +31,15 @@ edit_uri: ""
 
 plugins:
   - search
-  - awesome-pages
+  #  - mermaid2
+  - macros:
+      module_name: docs/macros
+      include_dir: ..
 
-google_analytics:
-  - UA-109858331-6
-  - auto
+extra:
+  analytics:
+    provider: google
+    property: G-290016612
```

Comment:
I'm not sure about that 🤔

Reply:
So am I. The previous solution wasn't working, so I tried using the same as in  Futures.